buildSrc/src/main/kotlin/Utilities.kt

```diff
@@ -123,8 +121,10 @@ fun isIntegrationsTestingEnabled() =
 fun Project.nessieClientForIceberg(): Dependency {
   val dependencyHandlerScope = DependencyHandlerScope.of(dependencies)
   if (!isIntegrationsTestingEnabled()) {
+    val clientVersion = libsRequiredVersion("nessieClientVersion")
     return dependencies.create(
-      "org.projectnessie:nessie-client:${libsRequiredVersion("nessieClientVersion")}"
+      if (clientVersion >= "0.50.1") "org.projectnessie.nessie:nessie-client:$clientVersion"
```

Comment:
Does `>=` just work here? Will it work for `0.100.0`?

However, chances are by the time we get to 0.100 we won't have to support 0.50 and earlier :thinking:

Reply:
0.100 is quite some time away. But yes, otherwise it would be bad.